src/lua/httpc.lua

```diff
@@ -32,8 +32,20 @@
 local driver = package.loaded.http.client
```

Comment:
Regarding the documentation request from the 1st commit:

> Be careful with defining his own encoding function - header with content
> type could contain content type like "text/html" as well as a header
> options [1], like "text/html; charset=UTF-8". Content type is passed to
> encoder function *with* options.

There may be a confusion: a key in the encoders table should miss the options, but the content type in the encoder function argument contains them. Technically your description is unambiguous, but maybe it has sense to highlight the difference somehow.

Feel free to ignore, it is just my thought around.